README.txt

Look at the manual and the Changelog file for information!


ChangeLog
Changelog

2013-10-02 - Update by Clemens Riccabona <www.riccabona.it>
* fixed issues: #44405, #29511, #9645, #11263
* updated compatibility in emconf, refactored complete additionalMarker hook

2013-10-01 - Update by Clemens Riccabona <www.riccabona.it>
* #11263
* formatting and changes to meet the CGL in various files.

2013-09-27 - Update by Clemens Riccabona <www.riccabona.it>
* Resolved bug: #46335, #48025
* changed mail-adress of Clemens Riccabona in em_conf
* obsolete require_once of pi_base in class pi1 removed

2013-04-23 - Update by Clemens Riccabona <www.riccabona.it>
* Resolved bugs: #9219, #28381, #8232, #31867

2010-06-14 Philip Almeida <philip.almeida@freedomson.com>
* HideIfInPast works with interval news items
* IE8 Tip Compatibility switch
* http://forge.typo3.org/issues/show/7266 - Add option - Thanks to Lukas Taferner
* Tutorial Update
* Add setup option "arrowLeftDisabled" - Thanks to Gregory Goidin
* Add a new marker for tt_news "NEWS_STARTENDATE" - Thanks to Gregory Goidin

2010-02-16 Philip Almeida <philip.almeida@freedomson.com>

*   Tutorial update
*   News category selector
    Sponsored by phase2-networks.com
    http://forge.typo3.org/issues/show/6356

2010-02-14 Philip Almeida <philip.almeida@freedomson.com>
Firefox 3.6 Compatible
Sponsored by digital-park.de
http://forge.typo3.org/issues/show/6304

2010-02-02
dam_ttnews compatible, no extra condiguration needed
sponsored by phase2-networks.com

2009-12-19
Added italian translation, thanks to ALEX TUVERI
2009-12-04
http://forge.typo3.org/issues/show/5585
http://forge.typo3.org/issues/show/5578
http://forge.typo3.org/issues/show/5524
Fixed render multiple dates template issue.

2009-11-24 - Update by Philip Almeida
IE google canvas API include, thanks to Dirk Gabler for the report.

2009-11-17 – Update by Philip Almeida
Altered tooltip render engine
Beauty Tips - http://www.lullabot.com/files/bt/bt-latest/DEMO/index.html
Tutorial updated to current template
Version 2.0.0

2009-11-08 – Update by Philip Almeida
Altered tooltip render engine
Now using: http://flowplayer.org/tools/tooltip.html

2009-11-01 - Update by Philip Almeida
Mouse over revised
Thanks to Andreas Döring
# Link to external page and article are now working, link to blank page on external url also enabled.
http://bugs.typo3.org/view.php?id=11928
# Javascript error removed.
http://bugs.typo3.org/bug_update_page.php?bug_id=11893
#   Translation enabled for "News Calendar" and "System Options" tabs.
Thanks to Sébastien DELCROIX

2009-09-12 - Update by Philip Almeida
Thanks to all bug reporters just possible because of you :)
# First Date Selector Add On - Beta - Thanks to Максим Левицкий - Maxim Levicky
# Better contextmenu positioning after mouseover 
# http://bugs.typo3.org/view.php?id=10709 (Thanks to Philipp Zbrug)
# http://bugs.typo3.org/view.php?id=11323
# http://bugs.typo3.org/view.php?id=10414
# http://bugs.typo3.org/view.php?id=8810
# http://bugs.typo3.org/view.php?id=8388
 # http://bugs.typo3.org/view.php?id=11484 (Tutorial Update)

2009-04-15 - Update by Clemens Riccabona
# Small code cleanup (if-statement without parentheses is not following CGL)
# Bugfix: if hideIfInPast was set in listView the entries were not displayed but the date-header; now resolved
# Some small, almost cosmetic changes in manual

2009-04-15 - Update by Clemens Riccabona
# locale_all setting extended by the global config.renderCharset (TypoScript - config) due to encoding bugs
# functions converSpecialCharacters: changed from hardcoded charset to config.renderCharset (TypoScript - config) due to encoding bugs
# Small array counter mistake
# Small bugfix -> sometimes empty first day of month appeared in calender listview

2009-01-24 Corrected bug reported by Jordan Bergen.

2009-01-02 Update by Michael Hitzler, paravista media: Use Hook in tt_news in order to display markers for start and end date.

2008-12-30 Added support for all news record types (internal page, external record).
Thanks to Gregory for reporting missing feature.

2008-05-30: Changes by Clemens Riccabona, Riccabona eSolutions: Added the possibilty to switch the JavaScript via TypoScript. Default is now a compressed version. Small issue with onmouseover event changed in rightcontext.js (and compressed version). Some German Translation changed/added.

2008-04-24 Date format on multiple events bug fixed.

2008-04-24 Special Characters bug fixed

2008-04-24 Cache bug fixed

2008-04-24 Changes by Clemens Riccabona, Riccabona eSolutions: Two new markers in template possible (but not set for backwards compatibility) for displaying image and short/bodytext (cropped) in listView. Added the possibility to hide past events on listView and calendar separately. All with full TypoScript support. Changed some internal file handling for more comfort and compatibility. Added German translations for the Frontend.

2008-03-25 Function gmmktime() removed in favor of mktime(); Day names in calendar are now parsed via Typoscript config option. Add new option to parsing function: “skip” .

2008-03-20 Add a flag to set the php special characters parser function. Thanks to Василий Мосийчук (Vasyl Mosiychuk ) - typo3.net.ua, for reporting and helping on the debug.

2008-03-18 Re – add Dutch translation missing feature since 2008-01-25.  Thanks to Jordan Bergen (GeefGratis.nl)

2008-03-14 XML compliance update thanks to Markus Waskowski (growing-media.de)

2008-02-04 Calendar table closing TR, thanks to Tomaz Zaman. 
		   Link URL rewrite.

2008-02-01 Fixed bug in strtotime conversion with german characters – Thanks to Markus Waskowski.
		  Fixed bug in display type “list”  datetime was not showing correctly – Thanks to James Huang 
		  Altered the CSS include render type (now renders inside the header tag) – Thanks to Marcus Persson 

2008-01-26 Possible to render plugin via Typoscript


ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext "newscalendar".
 *
 * Auto generated 27-09-2013 16:38
 *
 * Manual updates:
 * Only the data in the array - everything else is removed by next
 * writing. "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = array(
	'title' => 'News Calendar',
	'description' => 'Calendar for Ext:tt_news. Provides calendar and a list view functionality. Main sponsors: geefgratis.nl, eventonizer.nl, phase2-networks.com, digital-park.de',
	'category' => 'plugin',
	'shy' => 0,
	'version' => '2.1.8',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'module' => '',
	'state' => 'stable',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => 'tt_news',
	'clearcacheonload' => 0,
	'lockType' => '',
	'author' => 'Philip Almeida, Clemens Riccabona',
	'author_email' => 'philip.almeida@freedomson.com, clemens-at-riccabona{dot}it',
	'author_company' => 'freedomson.com riccabona.it',
	'CGLcompliance' => '',
	'CGLcompliance_note' => '',
	'constraints' => array(
		'depends' => array(
			'php' => '5.2.0-5.3.99',
			'typo3' => '4.5.0-6.1.99',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:46:{s:9:"ChangeLog";s:4:"b279";s:12:"ext_icon.gif";s:4:"f04e";s:17:"ext_localconf.php";s:4:"5708";s:14:"ext_tables.php";s:4:"77b3";s:14:"ext_tables.sql";s:4:"d348";s:15:"flexform_ds.xml";s:4:"2298";s:13:"locallang.xml";s:4:"9270";s:16:"locallang_db.xml";s:4:"ed1b";s:10:"README.txt";s:4:"7573";s:14:"doc/manual.sxw";s:4:"e3af";s:19:"doc/wizard_form.dat";s:4:"e21a";s:20:"doc/wizard_form.html";s:4:"5f71";s:49:"hooks/class.tx_newscalendar_additionalMarkers.php";s:4:"9cc2";s:14:"pi1/ce_wiz.gif";s:4:"f04e";s:33:"pi1/class.tx_newscalendar_pi1.php";s:4:"4167";s:41:"pi1/class.tx_newscalendar_pi1_wizicon.php";s:4:"2f2e";s:13:"pi1/clear.gif";s:4:"cc11";s:17:"pi1/locallang.xml";s:4:"3c94";s:24:"pi1/static/editorcfg.txt";s:4:"c760";s:14:"res/accept.png";s:4:"8bfe";s:29:"res/calendarViewTemplate.html";s:4:"23ab";s:14:"res/cancel.png";s:4:"757a";s:25:"res/class.newscalendar.js";s:4:"c3d3";s:19:"res/cssCalendar.css";s:4:"cb3b";s:22:"res/cssContextMenu.css";s:4:"092e";s:19:"res/date_changer.js";s:4:"69b9";s:17:"res/jquery.min.js";s:4:"bb38";s:25:"res/listViewTemplate.html";s:4:"ecd9";s:30:"res/bt-0.9.5-rc1/CHANGELOG.txt";s:4:"86bf";s:29:"res/bt-0.9.5-rc1/jquery.bt.js";s:4:"6f82";s:33:"res/bt-0.9.5-rc1/jquery.bt.min.js";s:4:"b988";s:48:"res/bt-0.9.5-rc1/other_libs/jquery.easing.1.3.js";s:4:"6516";s:58:"res/bt-0.9.5-rc1/other_libs/jquery.hoverIntent.minified.js";s:4:"bba7";s:56:"res/bt-0.9.5-rc1/other_libs/bgiframe_2.1.1/ChangeLog.txt";s:4:"0a88";s:61:"res/bt-0.9.5-rc1/other_libs/bgiframe_2.1.1/jquery.bgiframe.js";s:4:"880b";s:65:"res/bt-0.9.5-rc1/other_libs/bgiframe_2.1.1/jquery.bgiframe.min.js";s:4:"a868";s:66:"res/bt-0.9.5-rc1/other_libs/bgiframe_2.1.1/jquery.bgiframe.pack.js";s:4:"7b5a";s:52:"res/bt-0.9.5-rc1/other_libs/bgiframe_2.1.1/META.json";s:4:"bd9b";s:47:"res/bt-0.9.5-rc1/other_libs/excanvas_r3/AUTHORS";s:4:"e1b4";s:47:"res/bt-0.9.5-rc1/other_libs/excanvas_r3/COPYING";s:4:"3b83";s:51:"res/bt-0.9.5-rc1/other_libs/excanvas_r3/excanvas.js";s:4:"13b2";s:46:"res/bt-0.9.5-rc1/other_libs/excanvas_r3/README";s:4:"b7e6";s:22:"res/pic/arrow-left.png";s:4:"9ccb";s:23:"res/pic/arrow-right.png";s:4:"6a8f";s:20:"static/constants.txt";s:4:"78bf";s:16:"static/setup.txt";s:4:"fbd7";}',
	'suggests' => array(
	),
);

?>